src/controllers/lyrics.js

```diff
@@ -0,0 +1,236 @@
+import { appRouter } from '../components/router/appRouter';
+import { playbackManager } from '../components/playback/playbackmanager';
+import ServerConnections from '../components/ServerConnections';
+
+import LibraryMenu from '../scripts/libraryMenu';
+import globalize from '../scripts/globalize';
+import Events from '../utils/events.ts';
+
+import '../styles/lyrics.scss';
+
+let currentPlayer;
+let currentItem;
+
+let savedLyrics;
+let isDynamicLyric = false;
+
+function dynamicLyricHtmlReducer(htmlAccumulator, lyric, index) {
+    htmlAccumulator += `<div class="lyrics dynamicLyric" id="lyricPosition${index}" data-lyrictime="${lyric.Start}">${lyric.Text}</div>`;
```

Comment:
Again, shouldn't we escape HTML `lyric.Text`? Here and below.

Reply:
fixed on [a1768dd](https://github.com/jellyfin/jellyfin-web/pull/4733/commits/a1768dd4df99aaa428d8ef4ebc03c4d288e837fc)